### ISSUE TITLE
jump and hi-jump doing 0 dmg

### ISSUE THREAD

**Issue by alioh7** [EdenServer/community#2925]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details
was in a exp party as sam/drg fighting greater colibri and jump and hi-jump started doing 0 dmg as shown in the image below , no HP decrease, no TP gain and 0 dmg on both log and on screen.
this has happened before on a regular basis when i play sam/drg and once it happens all jumps and hi-jumps from this point do 0 dmg/no tp etc. 
![Aliohxtrm_2020 06 04_191541](https://user-images.githubusercontent.com/55878362/83802529-9b1dce00-a6b3-11ea-91fe-ad1a5e1a85c7.png)
![Aliohxtrm_2020 06 04_191701](https://user-images.githubusercontent.com/55878362/83802599-b557ac00-a6b3-11ea-87dc-9148c05892d5.png)

**Comment by perkuprightmeow** [comment]
This only happens when you’re dragoon sub? I tried to recreate the issue with DRG main but couldn’t. 
What weapon are you using ?

**Comment by Slamm21** [comment]
I wonder if this is related to this by any chance? had this happen twice in the last week or so.
![image](https://user-images.githubusercontent.com/5552081/83822869-34f07580-a6ca-11ea-8027-1e8931dedd17.png)

**Comment by alioh7** [comment]
yes so far it only happened as drg sub, when i was leveling dragoon main from 10-37 it never happened not once.
